src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSCloud.java

```diff
@@ -168,8 +168,8 @@ public String getInstanceCapStr() {
             return String.valueOf(instanceCap);
     }
 
-    public int getInstanceCap() {
```

Comment:
Yes. It is covered by `getInstanceCapStr`. The method I intended to add on Friday was the one in this PR but I selected the incorrect field from the insert getter dialog.